### PR TITLE
Updated with parameters for config

### PR DESCRIPTION
**Pull request by andyk-ms**
Eliminated pipeline variable
Updated with parameters on run config, allow manual run with clearing service resources only (no test)
Default with full run from scheduled trigger

### CODE REVIEW

File: vsts/vsts.yaml [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+
 resources:
   - repo: self
     clean: true
 
 jobs:
   ### Linux build ###
   - job: LINUX
-    displayName: Linux
+    displayName: Linux (timeout=${{ parameters.jobTimeoutInMinutes }}mins)
```
Comment (drwill-ms) [inline vsts/vsts.yaml:27]: Seems odd to make that one property part of the title.

Comment (drwill-ms) [reply vsts/vsts.yaml:27]: If we need this printed somewhere, I'd just document it in a step.

**Review by drwill-ms — APPROVED** [review]

File: vsts/vsts.yaml (hunk 1 above)
Comment (andyk-ms) [reply vsts/vsts.yaml:27]: I thought it allows us to see if the parameter is set, seeing the timeout knowing if custom parameter is taken on the job.
Alternative is to print this out on the log. We can change it if this info is not helpful.

**Comment by andyk-ms** [comment]
Updated the pipeline yaml with parameters.

**Pull request merged**